msgraph-connector-product/products/msgraph-calendar/product.json

```diff
@@ -0,0 +1,26 @@
+{
+  "installers": [
+    {
+      "id": "maven-dependency",
```

Comment:
I think we need an additional 'maven-import' installer, for the demo itself 
... similar as you see it here: https://github.com/axonivy-market/docusign-connector/blob/master/docusign-connector-product/product.json